src/main/java/com/boomi/flow/services/boomi/mdh/records/GoldenRecordRepository.java

```diff
@@ -39,19 +39,31 @@ public void delete(ApplicationConfiguration configuration, String universeId, Li
 
         // TODO: Cleanup everything in this filter block cause it's super ugly
         if (filter != null) {
+            GoldenRecordQueryRequest.Sort sort = new GoldenRecordQueryRequest.Sort();
+
+            // Add component filter to sort
+            if (filter.hasOrderByPropertyDeveloperName() && filter.getOrderByDirectionType() != null) {
+                sort.getFields().add(new GoldenRecordQueryRequest.Sort.Field()
+                        .setFieldId(filter.getOrderByPropertyDeveloperName().toUpperCase())
+                        .setDirection(filter.getOrderByDirectionType().toString())
+                );
+            }
 
+            // Add metadata orderBy[] to sort
```

Comment:
I understand what this comment means but it could be confusing in the future. 
we don't know when orderBy will be supported in the tooling also the other sort "orderByPropertyDeveloperName" could be added using a metadata modification

Reply:
I'll get rid of these comment, they don't really bring much value.